### PR TITLE
Example parallel test on Kobiton devices by using Katalon tool

### PR DESCRIPTION
**Pull request by tuyetngo**

### CODE REVIEW

**Comment by tuyetngo** [comment]
@khanhdodang @duylam please help to review it.

**Comment by tuyetngo** [comment]
@khanhdodang @duylam please help to merge it.

File: Parallel_test_on_Kobiton_devices_by_using_Katalon/GlobalVariables.glbl [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (duylam) [inline Parallel_test_on_Kobiton_devices_by_using_Katalon/GlobalVariables.glbl]: Should organize files as

```
|- java
|- katalon
       |- README.md <-- put the guideline here
       |- ... <-- other files 
```

The README should give:

- One or two statements for introducing Katalon and how to install it (link to its homepage)
- How to register trial account (link to Khanh Do post on blog site)
- How to run this project: update api key, replace device name, etc.

File: Parallel_test_on_Kobiton_devices_by_using_Katalon/bin/Object Repository/Heroku_web/form_login.rs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (duylam) [inline Parallel_test_on_Kobiton_devices_by_using_Katalon/bin/Object Repository/Heroku_web/form_login.rs]: look like all files under `bin` are generated, is it ok to remove them ?

File: Parallel_test_on_Kobiton_devices_by_using_Katalon/GlobalVariables.glbl (hunk 1 above)
Comment (tuyetngo) [reply Parallel_test_on_Kobiton_devices_by_using_Katalon/GlobalVariables.glbl]: ok a

File: Parallel_test_on_Kobiton_devices_by_using_Katalon/bin/Object Repository/Heroku_web/form_login.rs (hunk 2 above)
Comment (tuyetngo) [reply Parallel_test_on_Kobiton_devices_by_using_Katalon/bin/Object Repository/Heroku_web/form_login.rs]: I asked a member in Katalon team, those files should not be removed.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
-# Kobiton Samples
-This repository includes examples to demonstrate how to execute web/application Automation testing using [Kobiton](https://kobiton.com/) (**real devices** mobile cloud platform) in different programming languages.
+# Run automation test on Kobiton devices using Katalon tool
```
Comment (duylam) [inline README.md]: cc. @thaitrongnghi 

This README file is for all samples, should keep it as before. My suggestion was to put everything on this PR under a separate folder, below is my suggested structure

```
|- README <-- this file
|- Java
|- JavaScript <-- existing folders
|- Frameworks
       |- robot <-- this is for Nghi
       |- Katalon
               |- README
               |- ...
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
...
+
+## I. Katalon Studio tool
+   Katalon studio provides project templates for organizing test cases; fully supports Web, Android, iOS and API testing on all operating systems; runs/Debug test cases or test suites using multiple configurations and data sets and easy to integrate with Jenkins, GIT, JIRA, and qTest with native plugins.
+   Refer to [Katalon Document] (https://docs.katalon.com/display/KD) to install and start using Kalaton Studio.
```
Comment (duylam) [inline README.md]: `Refer to [Katalon Document](https://docs.katalon.com/di` <-- remove space

File: README.md [hunk 5]
```diff
@@ -1,2 +1,17 @@
...
+   Kobiton is a powerful platform that lets you create your perfect mobile test cloud. Sign up for a Free Trial and Start Testing in Minutes here: [Register Trial account](https://kobiton.com/freetrial/)
+
+## III. Run automation test on Kobiton devices using Katalon tool
+- Set up environment
+  + Get Api keys in [Settings API keys] (https://portal.kobiton.com/settings/keys)
+  + Integrate Katalon with Kobiton server
+- Mark favorite device on the Portal
+- Create testcase and test suite
+- Execute test
+- View report
```
Comment (duylam) [inline README.md]: Remove file `katalon/.DS_Store`

File: katalon/bin/.project [hunk 6]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (duylam) [inline katalon/bin/.project]: is it ok if we remove the `bin` folder , looks like it's generated folder

**Pull request closed without merging**